lms/static/js/courseware/credit_progress.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function() {
+    $('.detail-collapse').on('click', function() {
+        $('.requirement-container').toggleClass('is-hidden');
+        var el = $(this);
```

Comment:
Variables should be declared on the first line of a function, please move this up a line. Also, please follow the convention of names jQuery DOM element variables with $ as the first character ( so this would be $el ).